frontends/election-manager/src/screens/unconfigured_screen.tsx

```diff
@@ -229,19 +247,19 @@ export function UnconfiguredScreen(): JSX.Element {
     );
   }
 
-  if (isConvertSems && inputConversionFiles.length > 0) {
+  if (isUsingConverter && inputConversionFiles.length > 0) {
     return (
       <NavigationScreen mainChildCenter>
         <Prose textCenter>
-          <h1>Convert from SEMS files</h1>
+          <h1>Convert from {client?.getDisplayName()} files</h1>
```

Comment:
I noticed in the video that uploading the first file gives some nice visual feedback ("Loaded NH Card something something" in green) and then upon uploading the second file, there's no feedback and it flashes through one "Loading" screen and then the "Election loading..." modal. Two thoughts:
1. I think it would be nice to have confirmation of the second file and then get to click a button to start the actual processing of the files
2. Any way to remove the flashed "Loading" screen before the modal kicks in?

These suggestions may be out of scope but figured I'd raise them.

Reply:
These are definitely problems with the current implementation, but they existed before so I'm going to leave it as-is for now. However, I created https://github.com/votingworks/vxsuite/issues/1633 to ensure this feedback isn't lost.